.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -18,7 +18,13 @@ body:
       label: Installation Information
       description: Trigger the action "About", click on "Copy and close" and paste the output here
     validations:
-      required: true
+    required: true
+  - type: textarea
+    attributes:
+      label: Sourcegraph Cody + Code Search Plugin Version
+      description: Under Settings > Plugins, open the Sourcegraph Cody + Code Search plugin and copy the version number.
```

Comment:
@taylorsperry, what is the exact motivation behind adding this field?

Note that we have the plugin version already provided from the field above (`Installation Information`). E.g.:

```
IntelliJ IDEA 2023.3.2 (Ultimate Edition)
Build #IU-233.13135.103, built on December 20, 2023
Licensed to Virtus Lab sp. z o.o. / Mikołaj Kondratek
Subscription is active until November 30, 2024.
Runtime version: 17.0.9+7-b1087.9 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 10
Metal Rendering is ON
Non-Bundled Plugins:
  DevKit (233.13135.104)
  nl.rubensten.texifyidea (0.9.3)
  com.virtuslab.git-machete (4.1.1-27-SNAPSHOT+git.6ba3ca9-dirty)
  com.sourcegraph.jetbrains (6.0-localbuild) <- HERE
Kotlin: 233.13135.103-IJ
```

marked by "<- HERE"